sale_stock_operating_unit/models/stock_move.py

```diff
@@ -9,6 +9,7 @@
 class StockMove(models.Model):
     _inherit = 'stock.move'
 
+    @api.multi
```

Comment:
why not `api.model`?

Reply:
Because the original method is using a record in self:

https://github.com/odoo/odoo/blob/10.0/addons/stock/models/stock_move.py#L463

Is it better to call https://github.com/odoo/odoo/blob/10.0/addons/stock/models/stock_move.py#L451 directly?

Reply:
Delete first commit